Makefile

```diff
@@ -238,9 +238,17 @@ else ifeq ($(CHAIN),shyft)
 APP_LOAD_PARAMS += --path "44'/60'"
 DEFINES += CHAINID_UPCASE=\"SHYFT\" CHAINID_COINNAME=\"SHFT\" CHAIN_KIND=CHAIN_KIND_SHYFT CHAIN_ID=7341
 APPNAME = "Shyft"
+else ifeq ($(CHAIN),pulsechain)
+APP_LOAD_PARAMS += --path "44'/60'"
+DEFINES += CHAINID_UPCASE=\"PULSECHAIN\" CHAINID_COINNAME=\"PLS\" CHAIN_KIND=CHAIN_KIND_PULSECHAIN CHAIN_ID=369
+APPNAME = "PulseChain"
+else ifeq ($(CHAIN),pulsechain_testnet)
+APP_LOAD_PARAMS += --path "44'/60'"
+DEFINES += CHAINID_UPCASE=\"PULSECHAIN_TESTNET\" CHAINID_COINNAME=\"TPLS\" CHAIN_KIND=CHAIN_KIND_PULSECHAIN CHAIN_ID=940
```

Comment:
https://chainlist.org/ refers to PulseChain Testnet as "tPLS". Do you want to keep "TPLS", or use "tPLS"?